packages/mobile/App/services/sync/utils/preparedInsert.ts

```diff
@@ -0,0 +1,19 @@
+import { Repository } from "typeorm";
+import { DataToPersist } from "../types";
+
+/** 
+ * Much faster than typeorm bulk insert or save
+ * Prepare a raw query and execute it with the values
+ */
+export const preparedInsert = async (repository: Repository<any>, rows: DataToPersist[]) => {
+    const tableName = repository.metadata.tableName;
+    const columns = Object.keys(rows[0]);
```

Comment:
Are we certain all rows will contain the same columns?

Reply:
Was just wanting to make sure that every row has all column keys present, even if the data is undefined/null - I would assume that's how the lookup table works but I'm not 100% sure